<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewer</class>
 <widget class="QMainWindow" name="Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>701</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QScrollArea" name="scrollAreaLeft">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContentsLeft">
           <property name="geometry">
            <rect>
             <x>36</x>
             <y>95</y>
             <width>500</width>
             <height>380</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>380</height>
            </size>
           </property>
          </widget>
         </widget>
         <widget class="QScrollArea" name="scrollArea_Right">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContentsRight">
           <property name="geometry">
            <rect>
             <x>36</x>
             <y>95</y>
             <width>500</width>
             <height>380</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>380</height>
            </size>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QFrame" name="frameTools">
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QWidget" name="layoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>252</width>
            <height>841</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayoutInfo">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Diagnosis</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>65</height>
              </size>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.Helvetica Neue DeskInterface'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>AA age Death Rate DR.dr%</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_4">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>10</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>10</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetNoConstraint</enum>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Picture Zoom (e.g. 1:1K)</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineZoomEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>90</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>1:1K</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QFrame" name="frame_6">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>10</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>10</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Preparation Type
 </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Dye Type
 </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkPartBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>172</height>
                </size>
               </property>
               <property name="text">
                <string>Is it Part of Other Picture</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_3">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QFrame" name="frame_5">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>10</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>10</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="DefPrepButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Define Preparation Type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="DefDyeButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Define Dye Type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_7">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>10</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>10</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>Evidences list</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeWidget" name="treeWidget">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>16777215</height>
              </size>
             </property>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="newEvButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>New Evidence</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="chEvButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Change Evidence</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="remEvButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Remove Evidence</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_8">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>250</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
       <widget class="QListWidget" name="listWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomAuto"/>
   <addaction name="separator"/>
   <addaction name="actionArrow"/>
   <addaction name="actionRect"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionPolygon"/>
   <addaction name="separator"/>
   <addaction name="actionConfirm"/>
   <addaction name="actionClear"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionHide_ShowBottom"/>
   <addaction name="actionHide_ShowRight"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/open_image_file_32.png</normaloff>
     <normalon>:/open_image_file_32.png</normalon>:/icons/open_image_file_32.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/Plus_zoom_or_search_symbol_of_interface_32.png</normaloff>:/icons/Plus_zoom_or_search_symbol_of_interface_32.png</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/Zoom_out_32.png</normaloff>
     <normalon>:/Zoom_out_32.png</normalon>:/icons/Zoom_out_32.png</iconset>
   </property>
   <property name="text">
    <string>ZoomOut</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionZoomAuto">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/Search_32.png</normaloff>
     <normalon>:/Search_32.png</normalon>:/icons/Search_32.png</iconset>
   </property>
   <property name="text">
    <string>ZoomAuto</string>
   </property>
   <property name="toolTip">
    <string>Zoom Auto</string>
   </property>
  </action>
  <action name="actionArrow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/Arrow_pointing_to_right_32.png</normaloff>
     <normalon>:/Arrow_pointing_to_right_32.png</normalon>:/icons/Arrow_pointing_to_right_32.png</iconset>
   </property>
   <property name="text">
    <string>Arrow</string>
   </property>
   <property name="toolTip">
    <string>Arrow Selectoin Tool</string>
   </property>
  </action>
  <action name="actionRect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/Rectangular_shape_outline_32.png</normaloff>
     <normalon>:/Rectangular_shape_outline_32.png</normalon>:/icons/Rectangular_shape_outline_32.png</iconset>
   </property>
   <property name="text">
    <string>Rect</string>
   </property>
   <property name="toolTip">
    <string>Rectangle Selection Tool</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/Ellipse_outline_shape_variant_32.png</normaloff>
     <normalon>:/Ellipse_outline_shape_variant_32.png</normalon>:/icons/Ellipse_outline_shape_variant_32.png</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
   <property name="toolTip">
    <string>Ellipse Selection Tool</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/Pentagon__32.png</normaloff>
     <normalon>:/Pentagon__32.png</normalon>:/icons/Pentagon__32.png</iconset>
   </property>
   <property name="text">
    <string>Polygon</string>
   </property>
   <property name="toolTip">
    <string>Polygonal Selection Tool</string>
   </property>
  </action>
  <action name="actionConfirm">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/Check_mark_32.png</normaloff>
     <normalon>:/Check_mark_32.png</normalon>:/icons/Check_mark_32.png</iconset>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
   <property name="toolTip">
    <string>Confirm Selection</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/Save_Disk_32.png</normaloff>
     <normalon>:/Save_Disk_32.png</normalon>:/icons/Save_Disk_32.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save to Data Base</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/Cross_remove_sign_32.png</normaloff>
     <normalon>:/Cross_remove_sign_32.png</normalon>:/icons/Cross_remove_sign_32.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear Unconfirmed Selections</string>
   </property>
  </action>
  <action name="actionHide_ShowRight">
   <property name="icon">
    <iconset>
     <normalon>:/icons/Right_arrow_fold_button_32.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Hide/ShowRight</string>
   </property>
   <property name="toolTip">
    <string>Hide/ShowRight</string>
   </property>
  </action>
  <action name="actionHide_ShowBottom">
   <property name="icon">
    <iconset>
     <normalon>:/icons/Down_arrow_fold_button_32.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Hide/ShowBottom</string>
   </property>
   <property name="toolTip">
    <string>Hide/ShowBottom</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections/>
</ui>
